girder/stemserver_plugin/stemimage.py

```diff
@@ -14,10 +14,9 @@ def __init__(self):
         super(StemImage, self).__init__()
         self.resourceName = 'stem_images'
         self.route('GET', (), self.find)
-        self.route('GET', (':id', 'bright'), self.bright)
-        self.route('GET', (':id', 'dark'), self.dark)
-        self.route('GET', (':id', 'bright', 'shape'), self.bright_shape)
-        self.route('GET', (':id', 'dark', 'shape'), self.dark_shape)
+        self.route('GET', (':id', 'names'), self.image_names)
+        self.route('GET', (':id', ':name'), self.image)
```

Comment:
Do we now require that images to have a name in the HDF5 file?

Reply:
Maybe. I could also add an endpoint `self.route('GET', (':id', ':index'), self.image)`, so that they can be accessed by index instead.

Reply:
That would be the same endpoint :-) I think we can just allow `name` to be a index as well. So we would try to convert name to an int first and if that successes use that?

Reply:
Sure! I guess we just have to make sure we don't name the images with numbers...